floats.go

```diff
@@ -590,6 +590,13 @@ func Prod(s []float64) float64 {
 	return prod
 }
 
+// Reverse reverses the order of elements in the slice.
+func Reverse(s []float64) {
+	for i, j := 0, len(s)-1; i < j; i, j = i+1, j-1 {
```

Comment:
Not mine. This was one of the early examples in Miek Gieben's Learning Go book which was the first book I read (I think possibly the first book on Go).